packages/turf-rhumb-destination/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@turf/destination",
```

Comment:
@stebogit Here's your problem 🤦‍♂️  You've named it `@turf/destination` and when `lerna boostrap` creates the symbolic links it overwrites the "real" `@turf/destination` with the rhumb module. I'll fix this and the tests should pass.